201901_gaussian_fg_lensed_cmb_realistic_noise/simulation.cfg

```diff
@@ -0,0 +1,77 @@
+
+output_nside = 4096
+output_folder = output/$output_nside
+channels = all
+unit = uK_CMB
+mapsims_version = "0.1"
+so_pysm_models_version = "0.1"
+
+[ other_components ]
+
+    [[ noise ]]
+
+    class = mapsims.SONoiseSimulator
+    nside = $output_nside
+    return_uK_CMB = True
+    sensitivity_mode = "baseline"
+    apply_beam_correction = True
+    apply_kludge_correction = True
+    scanning_strategy = "classical"
+    LA_number_LF = 1
+    LA_number_MF = 4
+    LA_number_UHF = 2
+    SA_years_LF = 1
+    SA_one_over_f_mode = "pessimistic"
+    seed = 8974
+
+[ pysm_components ]
+
+    # Remove this if no default PySM components are needed
+    pysm_components_string = "a1"
+
+    [[ dust ]]
+
+    class = so_pysm_models.GaussianDust
+    target_nside = $output_nside
+    has_polarization = True
+    TT_amplitude = 350.
+    Toffset = 18.
+    EE_amplitude = 100.
+    rTE = 0.35
+    EtoB = 0.5
+    alpha = -0.42
+    beta = 1.53
+    temp = 19.6
+    nu_0 = 353
+    seed = 20001
```

Comment:
See docs at https://so-pysm-models.readthedocs.io/en/latest/so_pysm_models/models.html#gaussiandust, default parameters by the Foregrounds group